src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.it.xlf

```diff
@@ -14,7 +14,7 @@
       </trans-unit>
       <trans-unit id="General_FastUpToDateCheck_LogLevel_Info">
         <source>Info</source>
-        <target state="translated">Informazioni</target>
+        <target state="translated">Info</target>
```

Comment:
Some changes to this string are converting the short form "Info" to the long translated form, others are shortening. Seems inconsistent.